Gems/LmbrCentral/Code/Source/Shape/TubeShape.cpp

```diff
@@ -379,6 +379,13 @@ namespace LmbrCentral
         const float radius, const AZ::u32 capSegments, const AZ::u32 sides,
         AZStd::vector<AZ::Vector3>& vertexBufferOut)
     {
+        if (const size_t segmentCount = spline->GetSegmentCount(); segmentCount == 0)
```

Comment:
nit: catching this to a variable seems unnecessary if it's not going to be used. This could just be:
`if (spline->GetSegmentCount() == 0)`

Reply:
Good shout will update 👍

Reply:
Wound up keeping the variable but then using it later in the function 👍